tfhe/src/high_level_api/keys/server.rs

```diff
@@ -351,6 +352,7 @@ impl Named for CompressedServerKey {
 pub struct CudaServerKey {
     pub(crate) key: Arc<IntegerCudaServerKey>,
     pub(crate) tag: Tag,
+    pub(crate) streams: CudaStreams,
```

Comment:
Is it ok to do that @tmontaigu? We won't be using with_thread_local_cuda_streams anymore? This looks strange. Also it looks strange that when we clone CudaStreams, new streams actually get created. @pdroalves I think we need to discuss this

Reply:
We still need that macro in some places, as within the clone trait. However, whenever we have a cuda server key that's where streams will be loaded from.